lib/wallaby/httpclient.ex

```diff
@@ -1,10 +1,13 @@
 defmodule Wallaby.HTTPClient do
+  @moduledoc """
+  Wallaby HTTPClient that handles making requests to the webdriver API
+  """
```

Comment:
This is actually an internal module that's not intended to be in our API docs. Would you mind changing this to `@moduledoc false`?

Reply:
👍  done